packages/vm/src/testing/storage.rs

```diff
@@ -47,8 +49,23 @@ impl ReadonlyStorage for MockStorage {
 
         let iter = self.data.range(bounds);
         Ok(match order {
-            Order::Ascending => Box::new(iter.map(clone_item).map(FfiResult::Ok)),
-            Order::Descending => Box::new(iter.rev().map(clone_item).map(FfiResult::Ok)),
+            Order::Ascending => Box::new(
+                iter.map(clone_item)
+                    .map(|item| {
+                        let len = (item.0.len() + item.1.len()) as u64;
```

Comment:
`len` -> `gas_cost` or smething like this

Reply:
fixed in 8c571ed